### ISSUE TITLE
Improve doc diffs between runs

### ISSUE THREAD

**Issue by sweetmandm** [smashingboxes/smashing_docs#15]
This is something that'd be great to see, and I found [this blog post while I was looking for a solution](https://spin.atomicobject.com/2012/04/27/intermittent-test-failures/). It sounds like this isn't actually a smashing_docs issue; I think it would likely the responsibility of the person writing the tests to make sure the results don't vary. If there are guidelines to make the results deterministic, it might be nice to add them to the readme.

I'm not aware of all the things that prevent the results from being deterministic, but I know these have been mentioned:

1. Randomized data. That blog post suggests factory_girl sequences:
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences

2. Dates change between runs. That blog post suggests timecop.

So basically, I'm wondering:
* Are there reasons those two suggestions above do not work or are not preferable
* Are there other things that cause unwanted noise between runs

**Comment by TylerRockwell** [comment]
Those suggestions may help to reduce noise between runs. The trouble, however, is that that's extra code that needs to be written just for the docs, and I don't know that developers will consistently want to exert the additional effort every time they write tests.

The other thing that comes to mind is that the Ids of factory data is randomized between runs, and I'm not sure how to control how this data is loaded into the database.

For now, I've added functionality to ease the process of uploading the `api_docs` to the wiki. Rather than manually having to do this every time, you essentially just add the wiki folder name to your config and flip a switch, and every time your docs are built, they get pushed to the wiki. This seems to be the path of least resistance for the moment, but if there's a better solution, I'm totally open to it.
